app/src/main/java/aashrai/android/gettowork/view/activity/SettingsActivity.java

```diff
@@ -49,6 +50,8 @@
   }
 
   private void configureSearch() {
+   search.setCompoundDrawables(null, null,
```

Comment:
Yes, the current implementation does work fine and that's why I didn't realise that but for the sake of keeping consistency, I guess we should use VectorDrawableCompat with referencing drawables with vector tags. What do you say?